saleor/payment/gateways/stripe/deprecated/tests/test_stripe.py

```diff
@@ -132,6 +132,19 @@ def test_authorize_without_capture(stripe_payment, sandbox_gateway_config):
     assert response.is_success is True
 
 
+@pytest.mark.integration
+@pytest.mark.vcr(filter_headers=["authorization"])
+def test_authorize_without_capture_for_not_partial(
```

Comment:
You add a test marked by `VCR` but I can't find a cassette for this test.

Reply:
OK, after discussion with Maciek, we wan't change deprecated stripe plugi, I removed all changes made in deprecated module.